Marlin/language_pl.h

```diff
@@ -159,13 +159,13 @@
 #define MSG_THERMAL_RUNAWAY                 "Zanik temp."
 #define MSG_ERR_MAXTEMP                     "Err: max. temp."
 #define MSG_ERR_MINTEMP                     "Err: min. temp."
-#define MSG_ERR_MAXTEMP_BED                 "Err: max. temp. loza"
-#define MSG_ERR_MINTEMP_BED                 "Err: min. temp. loza"
+#define MSG_ERR_MAXTEMP_BED                 "Err: max. temp. stolu"
+#define MSG_ERR_MINTEMP_BED                 "Err: min. temp. stolu"
```

Comment:
One character too long. Must be 20 characters or fewer. 18 or fewer is even better.